packages/avatar-group/src/vaadin-avatar-group-mixin.js

```diff
@@ -0,0 +1,535 @@
+/**
+ * @license
+ * Copyright (c) 2020 - 2024 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
+import { html, render } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { announce } from '@vaadin/a11y-base/src/announce.js';
+import { OverlayClassMixin } from '@vaadin/component-base/src/overlay-class-mixin.js';
+import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
+import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+
+const MINIMUM_DISPLAYED_AVATARS = 2;
+
+/**
+ * A mixin providing common avatar group functionality.
+ *
+ * @polymerMixin
+ * @mixes ResizeMixin
```

Comment:
Should this also have `@mixes OverlayClassMixin`?

Reply:
Good point, added.